test/maverick/consensus/state.go

```diff
@@ -1274,13 +1272,13 @@ func (cs *State) createProposalBlock() (block *types.Block, blockParts *types.Pa
 	}
 	proposerAddr := cs.privValidatorPubKey.Address()
 
-	message := cs.GetState().MakeHashMessage(cs.Round)
+	message := cs.state.MakeHashMessage(round)
```

Comment:
NOTE: Don't use `cs.GetState()` in receiver since it happens deadlock sometimes.